lib/http_negotiate.c

```diff
@@ -51,11 +51,11 @@ CURLcode Curl_input_negotiate(struct connectdata *conn, bool proxy,
   struct negotiatedata *neg_ctx;
 
   if(proxy) {
-    userp = conn->proxyuser;
-    passwdp = conn->proxypasswd;
+    userp = conn->http_proxy.user;
+    passwdp = conn->http_proxy.passwd;
     service = data->set.str[STRING_PROXY_SERVICE_NAME] ?
               data->set.str[STRING_PROXY_SERVICE_NAME] : "HTTP";
-    host = conn->proxy.name;
+    host = conn->http_proxy.name;
```

Comment:
CI builds failing, 
~~~
C:\projects\curl\lib\http_negotiate.c(58): error C2039: 'name' : is not a member of 'proxy_info' [C:\projects\curl\build.msvc2012\lib\libcurl.vcxproj]
          c:\projects\curl\lib\urldata.h(874) : see declaration of 'proxy_info'
~~~
should be host.name maybe?